gradle.properties

```diff
@@ -18,3 +18,15 @@ junit5.version=5.8.1
 protobuf.version=3.17.1
 slf4j.version=1.7.25
 jackson.version=2.12.4
+
+#version that is going to be updated automatically by releases
+version = 1.0.0
+
+#signing
+SIGNING_KEY_ID = 193EAE47
+SIGNING_PASSWORD = password
+SIGNING_KEY = ALDqwcArqwfsdqweqwrwr
```

Comment:
What is this value?  Is it a dummy value?  Or is it a secret?

Reply:
This is dummy value, real value is going to read at build time from [Github secrets values](https://github.com/substrait-io/substrait-java/blob/2ecc7c2f6f13ddfadd2368cd82346a7176761aa8/ci/release/README.md#publicprivate-key).